package.json

```diff
@@ -18,7 +18,9 @@
   },
   "license": "MIT",
   "author": "Firebase Team",
-  "files": ["lib"],
+  "files": [
+    "lib"
+  ],
```

Comment:
This file is formatted automatically by npm, e.g. when running `npm install`. Keeping the formatting different than the default one would result in this file being modified each time someone works on the project, so it might be a good idea to just stick to the default formatting.

Reply:
Oh I didn't realize it was formatted like this, then let's keep whatever npm wants :)